src/ui/auth/rackspace/rackspace.go

```diff
@@ -207,3 +226,18 @@ func randString() string {
 	}
 	return string(b)
 }
+
+// fake domain
+const defaultEmailDomain = "rackspace-mk8s.com"
```

Comment:
I think this default domain should include the word "fake" or something so it's very clear that it's not supposed to be functional.

Reply:
ok